x-pack/plugins/security/public/management/users/components/change_password_form/change_password_form.tsx

```diff
@@ -99,6 +100,7 @@ export class ChangePasswordForm extends Component<Props, State> {
           }
         >
           <EuiFieldText
+            autoComplete="off"
```

Comment:
```suggestion
            autocomplete="new-password"
```
We should treat this as a "new password" field, so browsers can react accordingly.
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete):

> **`"new-password"`**
> 
> A new password. When creating a new account or changing passwords, this should be used for an "Enter your new password" or "Confirm new password" field, as opposed to a general "Enter your current password" field that might be present. This may be used by the browser both to avoid accidentally filling in an existing password and to offer assistance in creating a secure password (see also Preventing autofilling with autocomplete="new-password").